### DISCUSSION TITLE
Improve search function

### DISCUSSION THREAD

**Discussion by louim** [outlandishideas/wpackagist#80] [Ideas]
This isn't really an issue, be a place to discuss what could be done to improve the search. Seeing issue like #76 means that we can maybe come up with solutions and way to make the search more efficient.

What do you guys think? Do you have ideas on what you would like to improve or what could be done to make the search more useful?

**Comment by lavoiesl** [comment]
To do that, we would need to store the name of the plugin, which is not currently supported.

**Comment by NoelLH** [comment]
Moved to a Discussion for now until we have a proposal for specifically what we'd save and where, and how we'd index it. Compared to stabilising the core product it doesn't seem to me like this is one of our biggest problems from a quick skim of the issues.
